runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -59,6 +59,7 @@ dependencies {
   shadow project(path: ":beam-model-pipeline", configuration: "shadow")
   shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
   shadow project(path: ":beam-sdks-java-extensions-google-cloud-platform-core", configuration: "shadow")
+  shadow project(path: ":beam-sdks-java-fn-execution", configuration: "shadow")
```

Comment:
I don't think you need to add this dependency here, the worker has its own build.gradle file and already depends on this.